src/main/java/Duke.java

```diff
@@ -1,10 +1,63 @@
+import java.util.*;
```

Comment:
Wildcard imports should be avoided so that readers at a glance can tell what are the stuff that you have imported.